app/models/tube_creation.rb

```diff
@@ -32,6 +32,6 @@ def create_children!
   end
 
   def record_creation_of_children
-    #    children.each { |child| parent.events.create_tube!(child_purpose, child, user) }
+    # No record is made by this model class.
```

Comment:
Hmm yeah I'm not sure I'm afraid.  I know we need to define the method because the base class (`AssetCreation`) calls it in `process_children` without defining it, but that's all I know.  I think perhaps we should implement a stub method in `AssetCreation` so it can be overridden by the classes that actually define some behaviour, but that it's optional to do so.